src/server/config/cmds.go

```diff
@@ -345,7 +350,7 @@ func Cmds() []*cobra.Command {
 		Short: "Manages the pachyderm config.",
 		Long:  "Gets/sets pachyderm config values.",
 	}
-	commands = append(commands, cmdutil.CreateDocsAlias(configDocs, "config", "^pachctl config "))
+	commands = append(commands, cmdutil.CreateDocsAlias(configDocs, "config", " config "))
```

Comment:
I don't really understand what `CreateDocsAlias` does, but should the second `" config "` have spaces around it?

Reply:
This adds all of the matching commands in the 'Associated Commands' section of the help.  Specifically this didn't work on my machine because this assumes the name of the executable to be `pachctl`, whereas for me the help is:

```
$ pachctl help config
Gets/sets pachyderm config values.

Associated Commands:
  C:\Users\grey\go\bin\pachctl.exe config delete context      Deletes a context.
  C:\Users\grey\go\bin\pachctl.exe config get active-context  Gets the currently active context.
  C:\Users\grey\go\bin\pachctl.exe config get context         Gets a context.
  C:\Users\grey\go\bin\pachctl.exe config get metrics         Gets whether metrics are enabled.
  C:\Users\grey\go\bin\pachctl.exe config list context        Lists contexts.
  C:\Users\grey\go\bin\pachctl.exe config set active-context  Sets the currently active context.
  C:\Users\grey\go\bin\pachctl.exe config set context         Set a context.
  C:\Users\grey\go\bin\pachctl.exe config set metrics         Sets whether metrics are enabled.
  C:\Users\grey\go\bin\pachctl.exe config update context      Updates a context.
```

Not that this situation is good, but it's _more_ correct.